EntAppProject/src/main/resources/templates/JBFindAJob.html

```diff
@@ -11,21 +11,21 @@
 </head>
 
 <body>
- 
+
 <h1><span>Job</span>Book</h1>
 
 <h2>Find a Job Near You</h2>
- 
+
 <form action= "/JBUserProfile">
 
 Location:
 <br>
-<input style=position:left; type="text" id="userBio">
-<br> 
+<input style="float:left;" type="text" id="userBio">

```

Comment:
Same comment as above about style attributes needing quotes, and position: left not being a thing